PurchasesTests/Purchasing/OfferingsTests.swift

```diff
@@ -143,9 +143,9 @@ class OfferingsTests: XCTestCase {
         ])
 
         expect(offerings).toNot(beNil())
-        expect(offerings!["offering_a"]).toNot(beNil())
-        expect(offerings!["offering_b"]).toNot(beNil())
-        expect(offerings!.current).to(be(offerings!["offering_a"]))
+        expect(offerings?["offering_a"]).toNot(beNil())
+        expect(offerings?["offering_b"]).toNot(beNil())
+        expect(offerings?.current).to(be(offerings?["offering_a"]))
```

Comment:
for these, we could do: 
```swift
let offerings = try XCTUnwrap(offerings)
expect(offerings["offering_a")...
```

Reply:
Oh nice yeah, that makes the test stop right away instead of getting a million "found nil" errors.

Reply:
Done.